runtime/src/runtime/policy.rs

```diff
@@ -358,30 +358,24 @@ impl ProofSet {
     /// Create a `ProofSet` for enabled `RegisteredPoStProof`s
     pub fn default_post_proofs() -> Self {
         let mut proofs = vec![false; REGISTERED_POST_PROOF_VARIANTS];
```

Comment:
We should be able to drop the size of this `REGISTERED_POST_PROOF_VARIANTS` field now.

Reply:
What do you mean exactly by dropping the size of `REGISTERED_POST_PROOF_VARIANTS`?

Reply:
I think the number can change from 15 to 10?

Reply:
You mean from 15 to 5, no? Let me know if I am mistaking, I might not have all the context here.

I have also created counterpart PR in the FVM repo: https://github.com/filecoin-project/ref-fvm/pull/1896

Reply:
Oh, nvm, I was wrong -- we can't reduce this number at all, because it needs to be indexed up to `StackedDRGWindow64GiBV1P1`, which is 15.

Reply:
Yes but once the `builtin-actors` repo is making use of FVM4 this should be possible (the `V1P1` proofs are indexed between from 0 to 4).

Reply:
Unfortunately, these numbers are consensus-critical, so we can't change them (easily).

Reply:
Got it!